deepchecks/core/reduce_classes.py

```diff
@@ -101,7 +101,7 @@ def feature_reduce(aggregation_method: str, value_per_feature: pd.Series, featur
                        score_name: str) -> Dict[str, float]:
         """Return an aggregated drift score based on aggregation method defined."""
         if feature_importance is not None:
-            feature_importance = np.array(feature_importance)[value_per_feature.notna().values]
+            feature_importance = feature_importance[value_per_feature.notna().values]
```

Comment:
Did this because for drift, for instance, feature_importance is returned as a list instead of an array, and wanted to enforce this behavior. Are you sure the places in the code where it's a list have been changed?